### ISSUE TITLE
[BUG]: Rpt_Aria_MissingFocusableChild - shouldn't trigger for menu popup

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#1402]
### Project

a11y checker engine

### Browser

Chrome

### Operating system

MacOS

### Description

See Carbon overflow menu example: https://react.carbondesignsystem.com/iframe.html?args=&id=components-overflowmenu--default&viewMode=story

An overflow menu disappears when it loses focus, so there is no requirement to be able to tab back to it. We currently ignore comboboxes that control popups. We should ignore popups that are controlled by buttons with `aria-controls` also. Not sure if it should also indicate popup or aria-exanded?

### Steps to reproduce

See description. The Carbon overflow does not currently have the aria-controls, but should be enough to build a quick testcase to confirm a fix.

**Comment by francinelucca** [comment]
Carbon issue: https://github.com/carbon-design-system/carbon/issues/12728

**Comment by shunguoy** [comment]
<img width="1489" alt="Screenshot 2023-04-24 at 2 06 19 PM" src="https://user-images.githubusercontent.com/18539498/234092259-5dc0fba0-544b-496d-bd41-f18588d6b59f.png">

**Comment by ErickRenteria** [comment]
merged

**Comment by francinelucca** [comment]
Thanks! is there a release schedule we can check to know when this will be available in the tool?
